react-app/src/components/comment.js

```diff
@@ -9,7 +9,7 @@ const Comment = ({comment}) => {
   return (
     <div className="comment">
       <a className="avatar">
-        <img src={"https://i.pravatar.cc/300?u=" + comment.user_id}/>
+      <img className="" src={comment.profile_icon + ".png"}/>
```

Comment:
I believe this needs className = "ui avatar image"

Reply:
Since it is within a "comment" div and an "avatar" link, the correct styling is applied. I believe adding "ui avatar image" would provide conflicting styling. The styling for a comment avatar is slightly different than a regular avatar. I will remove the className="" though